src/Components/Components.sln

```diff
@@ -1,24 +1,55 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28306.52
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.28414.68
```

Comment:
VS made lots of changes to this file. VS doing VS things.

We should all be using 16.0 anyway, it's going to be required for our tooling soon/now.